packages/client-react/src/client/components/ListView/utils.js

```diff
@@ -0,0 +1,114 @@
+import { findIndex } from 'lodash';
+
+export const
+  isDef = thing => thing !== undefined,
+
+  noop = _ => {},
```

Comment:
Just a question. Why a lot of people use `noop` instead of `() => {}` ?

[lodash noop source](https://github.com/lodash/lodash/blob/912d6b04a1f6b732508a6da72a95ec4f96bda154/lodash.js#L15733)

```
function noop() {
    // No operation performed.
}
```

[Benchmark](https://jsperf.com/lodash-noop/3)

![chrome-noop](https://user-images.githubusercontent.com/24652543/34553963-6e2811b0-f13b-11e7-91ee-fb7b023ef06e.png)

![firefox-noop](https://user-images.githubusercontent.com/24652543/34553970-76f0f230-f13b-11e7-9771-05cdf18018af.png)